README.rst

```diff
@@ -2,15 +2,31 @@
 gabbi-multipart
 ---------
 
-A content-handler for gabbi that can prepare multipart/form-data
-request bodies.
+A content-handler for gabbi that can handle multipart/form-data request bodies.
 
-A work in progress.
+Usage
+=====
 
-TODO
-====
+Add the handler to your test building code:
 
-* more testing
-* documentation
-* removal of redundancy in simple_wsgi
-* a better README
+.. code:: python
+
+  def load_tests(loader, tests, pattern):
+    test_dir = os.path.join(os.path.dirname(__file__), TESTS_DIR)
+    return driver.build_tests(test_dir, loader,
+                              host='site.com',
+                              content_handlers=[Multipart])
+
+Then write a test:
+
+.. code:: yaml
+
+  name: multipart test
+  POST: /somewhere
+  request_headers:
+    content-type: multipart/form-data
+    data:
```

Comment:
This is an indent error. `data` shouldn't be under `request_headers`, but instead at the same level.